### ISSUE TITLE
Update es_ES app translation for 3.10.0

### ISSUE THREAD

**Issue by pljones** [jamulussoftware/jamulus#3122]
Hi @ignotus666,
We are getting ready for the 3.10.0 release. No further changes to translatable strings are currently planned for this release.

We would be happy if you updated the Jamulus software translations for **es_ES** by **2023-08-20**.

Please either [update the translations on Hosted Weblate](https://hosted.weblate.org/projects/jamulus/) or use Git:
- Update your fork from `jamulussoftware/jamulus` `main` and create a working branch
- Update translations using Qt Linguist in your fork,
- Commit and push your changes and reference this Issue,
- Open a Pull Request before 2023-08-20

Suggested commit and PR message:
```
Update es_ES app translation for 3.10.0

Fixes #<Insert this issue's number here>
```

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/main/docs/TRANSLATING.md).

Thanks for contributing to Jamulus!

<a href="https://hosted.weblate.org/engage/jamulus/"><img src="https://hosted.weblate.org/widgets/jamulus/-/jamulus-app/multi-auto.svg" alt="Translation status" /></a>

**Comment by ann0see** [comment]
You already know that probably, but I messed up. @ignotus666 could you please check if the weblate PR #3119 is correct?

**Comment by ignotus666** [comment]
@ann0see looks good. But a bunch of stuff got lost from the website translation. Did this happen because an EN PR got merged before the Weblate stuff? When that happens isn't the easiest fix to just revert the offending PR, merge the Weblate PR, and then merge it? It's what I did on previous occasions and it wasn't too much hassle.

**Comment by ann0see** [comment]
> merge the Weblate PR, and then merge it? It's what I did on previous occasions and it wasn't too much hassle.

Yes. But there was no *new* weblate PR. The stuff was not yet committed on weblate.

**Comment by ignotus666** [comment]
Ah, I see.

**Comment by ignotus666** [comment]
I think I recall a discussion about preventing changes to EN files being merged if a weblate PR was present, but it still wouldn't protect against this scenario. Maybe a warning in the PR template to make sure everything has been committed, pushed and merged from Weblate first as a reminder?

**Comment by ann0see** [comment]
No. I think we'd need to manage the .po file creation differently.

**Comment by ignotus666** [comment]
I'm all ears, but I don't see how - I think it's a problem inherent to how Weblate works.

**Comment by ann0see** [comment]
Kind of. But there should be an easy way to fix conflicts- I can't connect to their repo. I believe the po file generation should be linked to the current state on Weblate. Maybe there's an API.

**Comment by ignotus666** [comment]
Not sure it has anything to do with it, but you have to make sure you connect to the right branch on Weblate's fork - I was looking at `next-release` and was really puzzled as it seems to be based on one of my testing branches from a year ago. The right one is [weblate-jamulus-1-index](https://github.com/weblate/jamuluswebsite/tree/weblate-jamulus-1-index).

**Comment by ann0see** [comment]
So is next-release wrong on jamulussoftware/jamuluswebsite???
